config/exclude_files.json

```diff
@@ -1 +1,96 @@
-{}
\ No newline at end of file
+{
+  "red-hat-data-services/data-science-pipelines-operator":{
```

Comment:
I believe we discussed taking out RBAC configs from snyk scans. If so, we could also add the `config/rbac` and `config/argo` directories here, would reduce 16 HL CVEs.